advanced-usage/index.md

```diff
@@ -368,46 +401,157 @@ The File Browser allows viewing, editing, downloading, and uploading BlueOS file
-- filter for particular messages
-- view past and current messages
-- click on messages to see their full details
+- The first time BlueOS starts up it will auto-configure any cameras that are 
+connected at that time, with UDP streams counting up from port `5600`
+   - e.g. a second camera at first startup would be streamed to port `5601`
+   - Auto-configuration also occurs if the settings are reset
+      - applies to both [global settings resets](#settings) (via the sidebar) and
+      - camera manager settings resets (via the settings icon in the bottom right)
+- After the initial startup, settings are saved and persistent across reboots
+   - Further changes require manually re-configuring streams
+   - New streams need to be manually added
+      - UDP stream endpoints should be set to `udp://<surface-IP>:<port>`
+         - e.g. `udp://192.168.2.1:5602`
+      - RTSP stream endpoints are auto-configured with appropriate values
+   - One video input can have multiple output streams by clicking the blue `+`
+   symbol during stream configuration
+      - This only works for streams of the same endpoint type (e.g. it is not
+      currently possible to mix UDP and RTSP output streams for the same input)
+
+{{ easy_image(src="video-stream-example", width=400, center=true) }}
+
+- The streams are also presented via MAVLink, so QGroundControl (>=v4.1.7) can
+toggle between them without needing to know specific ports.
+
+{{ easy_image(src="qgc_switch_streams", width=400, center=true) }}
+
+- Camera settings (brightness, exposure, etc) that are exposed via UVC can be
+configured with the "Configure" button
+
+{{ easy_image(src="video-config-example", width=500, center=true) }}
+
+- Switching streams in QGroundControl while recording stops the current recording
+   - If you are regularly switching streams it may be worth doing a screen recording
+   either instead of or as well as recording the base video
+- QGroundControl does not yet support displaying multiple streams simultaneously
+   - Additional streams can be processed/viewed/recorded by [the options discussed
+   here](https://discuss.bluerobotics.com/t/how-to-stream-another-cameras-video/9573/3#receiving-the-stream-7)
+       - Note that some playback applications (e.g. VLC)  treat odd-numbered ports
+       as audio channels, so relevant video streams should only use even-numbered
+       ports
+       - UDP streams have the option to download an SDP file to easier playback
+       on some applications `(New in 1.1)`
+- Raspberry Pi cameras are supported `(New in 1.1)`
+   - Detection requires turning on legacy camera support:
+      1. turn on via the settings button in the buttom right corner
+      2. reboot the onboard computer to enable
+{% pirate() %}
+- It is possible to use the "Redirect source" element to make an ethernet camera
+available via the BlueOS camera manager, which allows QGroundControl to detect
+it automatically (via MAVLink)
```

Comment:
we are also planning to automatically detect IP cameras via onvif and expose their streams to qgc.

Reply:
plans can get documented when they're implemented